### DISCUSSION TITLE
Is it possible to prevent certain pages from being indexed by the search?

### DISCUSSION THREAD

**Discussion by ChristianIvicevic** [shuding/nextra#1347] [Q&A]
I am using Nextra as a wiki for a game of mine and I have some features that are not yet public yet I'd like to prepare pages for those. Instead of creating separate branches for those I was curious whether I can just prevent certain pages from being indexed as they appear in the search in the top right if somebody guesses their contents/title.

**Comment by DaveKeehl** [comment]
I would also need this feature as I have a changelog page that pollutes the search results.
